public/js/verilog.js

```diff
@@ -255,7 +255,20 @@ verilog = {
         return res;
     },
     santizeLabel: function(name){
-        return name.replace(/ Inverse/g, "_inv").replace(/ /g , "_");
+//        return name.replace(/ Inverse/g, "_inv").replace(/ /g , "_");
+
+        var temp = name;
+        if (temp.search(/ /g) < temp.length-1 && temp.search(/ /g) >= 0) {
+            temp = temp.replace(/ Inverse/g, "_inv");
+            temp = temp.replace(/ /g , "_");
+        }
+
+        if (temp.substring(0,1).search(/\\/g) < 0) {
+            if (temp.search(/[\W]/g) > -1 || temp.substring(0,1).search(/[0-9]/g) > -1)
+                temp = "\\" + temp + " ";
+        }
+
+        return temp;
```

Comment:
@drjyeh Can you document this sanitization logic, so it will be helpful for future developers